dotenv/load_test.ts

```diff
@@ -29,7 +29,7 @@ Deno.test({
 });
 
 Deno.test({
-  name: "load when multiple files",
+  name: "load() loads multiple files",
```

Comment:
```suggestion
  name: "load() works as expected when the multiple files are imported",
```

ref: https://github.com/denoland/deno_std/pull/1964